### DISCUSSION TITLE
choice.html is having an unexpected PARTY

### DISCUSSION THREAD

**Discussion by zzeleznick** [johnlindquist/kit#115] [Errors] [answered]
## Background

There's a new `<h1>PARTY</h1>` element that was added in the latest [kitapp commit](
https://github.com/johnlindquist/kitapp/commit/342e812259d0373d6f3b2b86246d80841841493c#diff-26ad4b834941d9b19ebf9db8082bd202aaf72ea0ddea85f5a8a0cb3c729cc6f2R671) that appears in each displayed choice when `choice.html` is set.

## Example

From my `kit-discussions` script:

<img width="489" alt="PARTY" src="https://user-images.githubusercontent.com/5779832/113493248-80aeaf80-9492-11eb-80c6-d4c4ac2221d3.png">

**Comment by johnlindquist** [comment]
lol, whooooooops. I was playing with the idea of `choice.html` giving the user complete control of the visuals of the choice. I'll sort it out

**Comment by johnlindquist** [comment, marked as answer]
Sadly, the PARTY is over. The new build now supports `choice.html` overriding the entire choice "button". This is still in an "experimental" phase though. Here was my test case.

```js
//Shortcut: opt /

let value = await arg(
  "testing choice html",
  ["white", "red", "yellow"].map(color => ({
    name: color,
    value: color,
    html: `
    <style>.selected{color: blue}</style>
    <div style="background-color:${color};width:100%;height:100%">${color}</div>`,
  }))
)

await arg("The value is", value)
```
